internal/vendor/wiz.go

```diff
@@ -156,6 +176,7 @@ type ProjectCloudOrganizationLink struct {
 	CloudOrganization CloudOrganization `json:"cloudOrganization"`
 	Environment       string            `json:"environment"` // enum Environment
 	ResourceTags      []*ResourceTag    `json:"resourceTags"`
+	ResourceGroups    []string          `json:"resourceGroups"`
```

Comment:
This field is optional in the GraphQL schema.

```suggestion
	ResourceGroups    []string          `json:"resourceGroups,omitempty"`
```